examples/avatar_agents/tavus/agent_worker.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+
+from dotenv import load_dotenv
+
+from livekit.agents import (
+    Agent,
+    AgentSession,
+    JobContext,
+    RoomOutputOptions,
+    WorkerOptions,
+    WorkerType,
+    cli,
+)
+from livekit.plugins import openai, tavus
+
+logger = logging.getLogger("tavus-avatar-example")
+logger.setLevel(logging.INFO)
+
+load_dotenv()
+
+
+async def entrypoint(ctx: JobContext):
+    await ctx.connect()
+
+    session = AgentSession(
+        llm=openai.realtime.RealtimeModel(voice="alloy"),
+    )
+
+    persona_id = os.getenv("TAVUS_PERSONA_ID")
+    replica_id = os.getenv("TAVUS_REPLICA_ID")
+    tavus_avatar = tavus.AvatarSession(persona_id=persona_id, replica_id=replica_id)
+    await tavus_avatar.start(session, room=ctx.room)
+
+    await session.start(
+        agent=Agent(instructions="Talk to me!"),
+        room=ctx.room,
+        # TODO: audio is forwarded to the avatar, so we disable room audio output
+        room_output_options=RoomOutputOptions(audio_enabled=True),
```

Comment:
Should we ask Tavus to use our Datastream instead?

Reply:
It also appears to be less flexible, as the audio can't be transmitted faster than real-time.